src/components/SocialBtns/facebookSDK.js

```diff
@@ -0,0 +1,33 @@
+ /* eslint-disable */
```

Comment:
Maybe move to `src/services`? What do you think. It doesn't feel like its a component.